moped-editor/.env-cmdrc

```diff
@@ -2,16 +2,19 @@
   "local": {
     "REACT_APP_AWS_COGNITO_REGION": "us-east-1",
     "REACT_APP_AWS_COGNITO_USER_POOL_ID": "us-east-1_U2dzkxfTv",
-    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "3u9n9373e37v603tbp25gs5fdc"
+    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "3u9n9373e37v603tbp25gs5fdc",
+    "REACT_APP_HASURA_ENDPOINT": "https://coa-moped.herokuapp.com/v1/graphql"
   },
   "staging": {
     "REACT_APP_AWS_COGNITO_REGION": "us-east-1",
     "REACT_APP_AWS_COGNITO_USER_POOL_ID": "us-east-1_U2dzkxfTv",
-    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "3u9n9373e37v603tbp25gs5fdc"
+    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "3u9n9373e37v603tbp25gs5fdc",
+    "REACT_APP_HASURA_ENDPOINT": "https://coa-moped.herokuapp.com/v1/graphql"
   },
   "production": {
     "REACT_APP_AWS_COGNITO_REGION": "us-east-1",
     "REACT_APP_AWS_COGNITO_USER_POOL_ID": "us-east-1_Zc3pNWX51",
-    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "ins01e2a8d3vd8apvnd0jv10c"
+    "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "ins01e2a8d3vd8apvnd0jv10c",
+    "REACT_APP_HASURA_ENDPOINT": "https://coa-moped.herokuapp.com/v1/graphql"
   }
 }
```

Comment:
This is great, thank you. However, these are not likely to work once deployed. The start scripts package.json control which environment variables are loaded at runtime, and it seems a little different from the originals currently in production:

https://github.com/cityofaustin/atd-moped/blob/4fb212ceaee74ec2c4c6dcc2f6a0984fb2cb4c76/moped-editor/package.json#L66